app/templates/partials/forms/label.html

```diff
@@ -6,7 +6,7 @@
 
 <label {{attr|xmlattr}}>
   <div class="label__inner">
-    {{label.text}}
+    {{label.text|safe}}
```

Comment:
I think this line has broken a lot of the integration tests.
Eg. a test is checking for `Why doesn&#39;t Chewbacca receive a medal at the end of A New Hope?`
and it now returns `Why doesn\'t Chewbacca receive a medal at the end of A New Hope?`